### PR TITLE
optimus-manager: add page

### PR DESCRIPTION
**Pull request by rafay99-epic**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by rafay99-epic** [comment]
> LGTM, Thanks for your contribution.

Happy to contribute, brother. It's a tool that I use on a daily basis.

File: pages/linux/optimus-manager.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Optimus Manager
```
Comment (pixelcmtd) [inline pages/linux/optimus-manager.md:1]:
```suggestion
# optimus-manager
```

**Comment by rafay99-epic** [comment]
@pixelcmtd  I'm done with the changes; any other changes in which I can help

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**